wiki/Getting-Started-SwiftUI.md

```diff
@@ -0,0 +1,163 @@
+# Swift Package Manager with SwiftUI
+
+This guide will walk you through getting a [Workflow](https://wwt.github.io/SwiftCurrent/Classes/Workflow.html) up and running in a new iOS project.  If you would like to see an existing project, clone the repo and view the `SwiftUIExampleApp` scheme in `SwiftCurrent.xcworkspace`.
+
+The app in this guide is going to be very simple.  It consists of a view that will hold the [WorkflowView](https://wwt.github.io/SwiftCurrent/Structs/WorkflowView.html), a view to enter an email address, and an optional view for if your email contains `@wwt.com`.  Here is a preview of what the app will look like:
```

Comment:
I think we can riff on this verbiage some "...and an optional view for if your email contains `@wwt.com`."

Reply:
```suggestion
The app in this guide is going to be very simple.  It consists of a view that will hold the [WorkflowView](https://wwt.github.io/SwiftCurrent/Structs/WorkflowView.html), a view to enter an email address, and an optional view for when the user enters an email with `@wwt.com` in it.  Here is a preview of what the app will look like:
```

Reply:
```suggestion
The app in this guide is going to be very simple.  It consists of a view that will hold the [WorkflowView](https://wwt.github.io/SwiftCurrent/Structs/WorkflowView.html), a view to enter an email address, and an optional view for employees of WWT.  Here is a preview of what the app will look like:
```

Reply:
I went with my first suggestion and made the change in the other 2 starting guides.